boards/arm/stm32mp157c_dk2/doc/stm32mp157_dk2.rst

```diff
@@ -192,8 +203,11 @@ Serial Port
 
 The STM32MP157C-DK2 Discovery board has 8 U(S)ARTs.
 The Zephyr console output is assigned by default to the ram console to be dumped
-by the Linux Remoteproc Framework on Cortex®-A7 core. The UART 4 can enabled as
-Cortex®-M4 console.
+by the Linux Remoteproc Framework on Cortex®-A7 core. In order to keep the UART7
+free for future serial interactions with Arduino shield, the Zephyr UART console
+output is USART3 and is disabled by default. UART console can be enable through
+stm32mp157c_dk2_defconfig board file or prj.conf project files, and will disable
+existing ram console output. Default UART console settings are 115200 8N1.
```

Comment:
RAM is an acronym so should be upper case (and on line 205 too)

Reply:
Thank you for your feedback @dbkinder, this is now updated